### DISCUSSION TITLE
I set my email to show in public profile, but it doesn't take effect

### DISCUSSION THREAD

**Discussion by Dedicatus5457** [community/community#28575] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/78254776/183249373-dc575571-a829-49e3-8234-7b828210f0f9.png)
![image](https://user-images.githubusercontent.com/78254776/183249386-b06f15fe-e4f1-44fc-bbe5-4fb10278ce79.png)
![image](https://user-images.githubusercontent.com/78254776/183249402-2f3eed8a-4055-4951-b193-767302963814.png)

**Comment by airtower-luna** [comment]
I can see that Gmail address on your profile right now. Maybe it just needed a moment, or what you see is still in your browser cache?

**Comment by ChrisAcrobat** [comment]
>I can see that Gmail address on your profile right now. Maybe it just needed a moment, or what you see is still in your browser cache?

Same here.

**Comment by melvincwng** [comment, marked as answer]
@Dedicatus5457, email addresses are only visible to GitHub users who are logged in.

From your 3rd screenshot, I can see you are using 'Incognito mode' that probably doesn't have an account signed in, hence that's why you cannot see your email address.

And as others have mentioned, we are able to see your email address. So not to worry :)

Hope this clarifies, if you find this answer useful, you can kindly 'Mark as answer' so others know this issue is resolved. Take care.

**Comment by Dedicatus5457** [comment]
thanks for everyone answers!
